presto-common/src/main/java/com/facebook/presto/common/function/SqlFunctionProperties.java

```diff
@@ -155,6 +163,7 @@ public static Builder builder()
         private Locale sessionLocale;
         private String sessionUser;
         private Map<String, String> extraCredentials = emptyMap();
+        private JsonPathExtractionEngine jsonPathExtractionEngine;
```

Comment:
Can you default this to Presto , so that existing users need not modify this unless they want to.

Reply:
will default to DYNAMIC given your next comment